DateRangePicker/DateRangePickerView.swift

```diff
@@ -166,11 +166,16 @@ open class DateRangePickerView: NSControl, ExpandedDateRangePickerControllerDele
 	}
 	
 	// MARK: - Methods
+    open func makePopover() -> NSPopover {
```

Comment:
<img width="756" alt="screen shot 2016-11-11 at 1 48 14 pm" src="https://cloud.githubusercontent.com/assets/5813649/20228144/88e743c4-a815-11e6-996e-33b30f4661ea.png">
and this is probably why.. .standby.